source/adios2/CMakeLists.txt

```diff
@@ -66,14 +67,22 @@ if(ADIOS2_HAVE_SysVShMem)
   target_sources(adios2 PRIVATE toolkit/capsule/shmem/ShmSystemV.cpp)
 endif()
 
+if(ADIOS2_HAVE_ZeroMQ)
+  target_sources(adios2 PRIVATE
+    toolkit/transport/wan/WANZmq.cpp
+  )
+  target_link_libraries(adios2 PRIVATE ZeroMQ::ZMQ)
+endif()
+
 if(ADIOS2_HAVE_DataMan)
```

Comment:
Is DataMan still optional?  if not then the conditional check should be removed and just always add it.